packages/platform-core/API.md

```diff
@@ -103,6 +104,12 @@ export class ConfigurationControllerFactory {
     getInstance: (configFileName: LocalConfigurationFileName) => ConfigurationController;
 }
 
+// @public
+export class DefaultResourceNameValidator implements ResourceNameValidator {

```

Comment:
This impl should be internal to the backend package

Reply:
moved